src/app/core/services/add-update-delete-registration/add-update-delete-registration.service.ts

```diff
@@ -97,6 +102,11 @@ export class AddUpdateDeleteRegistrationService {
     const draftWithoutEmptyRegistrations = removeEmptyRegistrations(draft);
     const { registration } = await this.uploadAttachments(draftWithoutEmptyRegistrations);
 
+    // Track simple / complete obs registration
```

Comment:
Nå er det ikke mange linjene her, men dette er vel et brudd på [OnceAndOnlyOnce](http://wiki.c2.com/?OnceAndOnlyOnce)? ;-)

Reply:
Det er sant. Foretrekker du at det er i en funksjon eller noe sånt? Kan godt endre.

Reply:
Var mest for spøk, men funksjon er jo fint, da:)